gulpfile.js

```diff
@@ -95,7 +95,7 @@ var mappings = {
   },
   'batch.Management': {
     'dir': 'batchManagement/lib',
-    'source': 'arm-batch/2015-12-01/BatchManagement.json',
+    'source': 'arm-batch/2015-12-01/swagger/BatchManagement.json',
     'ft': 1
   }
 };
```

Comment:
Dont you want to generate using the version 0.16.0-Nightly20160317 where the requestContent issue was fixed.